core/modules/ckeditor5/js/ckeditor5.js

```diff
@@ -5,6 +5,12 @@
   Backdrop.editors.ckeditor5 = {
 
     attach: function (element, format) {
+      // Bail out if the editor already exists in the page; for example, when
+      // this attach method is called after an ajax request.
```

Comment:
This comment seems a bit misleading...

"if the editor already exists in the page" - there can be multiple editors on a page. It's rather the current element.

@argiepiano what do you think?

Reply:
@indigoxela, thanks. I changed it. LMK if this looks ok.